packages/core/src/Tab/Tab.tsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react'
+import * as Tabs from '@radix-ui/react-tabs'
+import { Text } from '../Text'
+import { TabContainer, TabList, TabTriggerChip, TabTriggerButton, TabTriggerRadio } from './Tab.styled'
+import { Box } from '../Box'
+import { Radio } from '../Radio'
+import { Label } from '../Label'
+
+export interface ITabsProps {
+  orientation?: 'horizontal' | 'vertical'
+  onHover?: boolean
+  onClick?: () => void
+  size?: string
+  tabsContent: {
+    id: string
```

Comment:
It'd be cool if `id` were optional and we auto-incremented it for each tab along with a UUID to avoid collisions if there are multiple `Tab` components rendered on the page, e.g. `asdas-tab1`, `asdas-tab2`, etc..

Reply:
that's a great idea, and maybe close to required considering the use case you provided